CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing guidelines
+
+## Formatting
+
+By default, formating is checked by gofmt tool, where almost format all the
+cases.
+
+The project does not have a line length limit, but the team try to be as small
+as possible, keeping the code clean on reading. Having an 80-120 char limit is
+an excellent number to split the line.
+
+## Style
+
+The project follows the Golang project guidelines that you can find in the
+following link:
+
+[https://github.com/golang/go/wiki/CodeReviewComments]()
+
+## Developer environment
+
+By default, all k4e projects rely on Makefiles, where users can run all the
+workflows operations.
+
+To make it easier, all Makefiles implement a help section to see the actions you
+can run. Here is an example:
+
+```
+--> make help
+Usage:
+  make <target>
+
+General
+  help             Display this help.
+
+Development
+  manifests        Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+  generate         Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+  fmt              Run go fmt against code.
+  vet              Run go vet against code.
+  test             Run tests.
+
```

Comment:
What about including `test-coverage` target as well ?
Would we like to have any remark regarding coverage percentage ?